### PR TITLE
2.0 preview

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Scroll listener moved to the template using `on-scroll` binding.

Moved addition of `no-padding` class in `ready` (avoid poking `classList` at each `attached`)

### CODE REVIEW

File: paper-dialog-scrollable.html [hunk 1]
```diff
@@ -149,12 +145,14 @@
 
     ready: function () {
       this._ensureTarget();
+      this.classList.add('no-padding');
```
Comment (bicknellr) [inline paper-dialog-scrollable.html:148]: Where is this class used?

File: paper-dialog-scrollable.html [hunk 2]
```diff
@@ -149,12 +145,14 @@
...
+      this._boundScroll = this._scroll.bind(this);
+      this.$.scrollable.addEventListener('scroll', this._boundScroll);
```
Comment (bicknellr) [inline paper-dialog-scrollable.html]: You could add `on-scroll="_scroll"` to `#scrollable` in the template.

File: paper-dialog-scrollable.html (hunk 1 above)
Comment (valdrinkoshi) [reply paper-dialog-scrollable.html:148]: this is used in here (lol) https://github.com/PolymerElements/paper-dialog-behavior/blob/master/paper-dialog-shared-styles.html#L51
We should move `paper-dialog-scrollable, paper-dialog, paper-dialog-behavior` into the same repo/package but that would be a breaking change (tracking issue here https://github.com/PolymerElements/paper-dialog-scrollable/issues/15)

Comment (bicknellr) [reply paper-dialog-scrollable.html:148]: [:+1:](https://github.com/PolymerElements/paper-dialog-scrollable/pull/54/files#diff-a6846e05f0c9cec2617a0dbad5ada8b2L155)

File: paper-dialog-scrollable.html (hunk 2 above)
Comment (valdrinkoshi) [reply paper-dialog-scrollable.html]: indeed! 💯 done 👌

File: bower.json [hunk 3]
```diff
@@ -21,18 +21,35 @@
...
+        "paper-dialog-behavior": "PolymerElements/paper-dialog-behavior#^1.0.0",
+        "paper-styles": "PolymerElements/paper-styles#^1.0.0",
+        "polymer": "Polymer/polymer#^1.0.0"
+      },
+      "devDependencies": {
+        "iron-component-page": "PolymerElements/iron-component-page#^1.0.0",
+        "iron-demo-helpers": "PolymerElements/iron-demo-helpers#^1.0.0",
+        "paper-button": "PolymerElements/paper-button#^1.0.0",
+        "paper-dialog": "PolymerElements/paper-dialog#^1.0.0",
+        "web-component-tester": "^5.0.0",
```
Comment (bicknellr) [inline bower.json:50]: This was `^4.0.0` before.

File: paper-dialog-scrollable.html (hunk 1 above)
Comment (bicknellr) [reply paper-dialog-scrollable.html:148]: Another question: why did this move from `attached` to `ready`?

Comment (valdrinkoshi) [reply paper-dialog-scrollable.html:148]: > Moved addition of no-padding class in ready (avoid poking classList at each attached)

**Review by bicknellr — APPROVED** [review]

**Pull request merged**